### ISSUE TITLE
Windows Summary Dashboard has two CPU Utilization widgets.

### ISSUE THREAD

**Issue by Grumblesmith** [netuitive-community-packages/netuitive-packages-windows#17]

**Comment by dgeorg42** [comment]
@Createor - I went back and looked at the original commit and even then there were two utilization widgets. One is Top 5 Highest Utilized (which is the one I typically add to the summary dashboards) and one is Utilization by Metric, which a) is configured to show CPU utilization across all elements (not just Top 5) and b) has a name that makes no sense. :)  I don't remember how/why the "Utilization by Metric" widget got on there, but I'd recommend removing it.

**Comment by Grumblesmith** [comment]
@dgeorg42 yes I noticed the same items regarding that widget. I suspect it should have been a top 5 memory widget like other default dashboards and intend to update it as such.
